src/MizarData/BaselineAndComparisonHelper.cpp

```diff
@@ -12,55 +12,59 @@
 
 #include "MizarBase/AbsoluteAddress.h"
 #include "MizarBase/SampledFunctionId.h"
+#include "MizarData/MizarDataProvider.h"
 
 using ::orbit_mizar_base::AbsoluteAddress;
 
 namespace orbit_mizar_data {
 
 using SFID = ::orbit_mizar_base::SFID;
 
-template <typename K, typename V>
-[[nodiscard]] static absl::flat_hash_set<V> ValueSet(const absl::flat_hash_map<K, V>& map) {
-  absl::flat_hash_set<V> result;
+[[nodiscard]] static absl::flat_hash_set<std::string> FunctionNamesSet(
+    absl::flat_hash_map<AbsoluteAddress, FunctionSymbol> map) {
+  absl::flat_hash_set<std::string> result;
   std::transform(std::begin(map), std::end(map), std::inserter(result, std::begin(result)),
```

Comment:
I just learned about `absl::c_transform`. This could be used here as well I guess. But also absolutely not necessary

Reply:
Already done.